service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/C21OrderEmailContentProvider.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import net.logstash.logback.encoder.org.apache.commons.lang3.ObjectUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.service.DateFormatterService;
+import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
+
+import java.time.format.FormatStyle;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.joining;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.CASE_TYPE;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.JURISDICTION;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildSubjectLine;
+
+@Service
+public class C21OrderEmailContentProvider extends AbstractEmailContentProvider {
+
+    private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
+    private final ObjectMapper objectMapper;
+
+    protected C21OrderEmailContentProvider(@Value("${ccd.ui.base.url}")String uiBaseUrl,
+                                           ObjectMapper objectMapper,
+                                           HearingBookingService hearingBookingService,
+                                           LocalAuthorityNameLookupConfiguration
+                                               localAuthorityNameLookupConfiguration,
+                                           DateFormatterService dateFormatterService) {
+        super(uiBaseUrl, dateFormatterService, hearingBookingService);
+        this.objectMapper = objectMapper;
+        this.localAuthorityNameLookupConfiguration = localAuthorityNameLookupConfiguration;
+    }
+
+    public Map<String, Object> buildC21OrderNotification(final CaseDetails caseDetails,
+                                                         final String localAuthorityCode) {
+        // Validation within our frontend ensures that the following data is present
```

Comment:
This comment is too generic, maybe would be better to move it above the line to which it applies?

Reply:
removed in [pr-618](https://github.com/hmcts/fpl-ccd-configuration/pull/618) as its a follow-up to this. Would you want to done here also?

Reply:
No it's fine only in PR-618